text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/tag/resolver/Formatter.java

```diff
@@ -139,4 +144,48 @@ public static TagResolver booleanChoice(@TagPattern final @NotNull String key, f
       return Tag.inserting(context.deserialize(value ? trueCase : falseCase));
     });
   }
+
+  /**
+   * Creates a replacement that inserts a list of components. These components are joined together by {@link Component#join(JoinConfiguration.Builder, ComponentLike...)}.
+   *
+   * <p>This tag expects a separator as the first argument. A last separator may optionally be provided.</p>
+   *
+   * <p>This replacement is auto-closing, so its style will not influence the style of following components.</p>
+   *
+   * @param key the key
+   * @param components the components to join
+   * @return the placeholder
+   * @since 4.14.0
```

Comment:
```suggestion
   * @since 4.15.0
```

4.14.0 has already been released.....